public/store/css/main.css

```diff
@@ -1135,6 +1135,16 @@ VOUCHER PAGES
 	background-color: #f8c28b;
 }
 
+.alert-message.success {
+	background-color: #dff0d8;
+    border: 1px solid #3c763d;
+	border-radius: .25rem;
+	display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+	margin: 1rem 10%;
```

Comment:
A bit lumpy indents. I'd like to standardise this file at some point, it's a mix of tabs and spaces right now.

flex-direction: row is the default and wrap won't be doing too much but they don't hurt. I'd probably make border-radius 3px to match the other border radii in the file.

Reply:
thank you! yes, please, let's standandardise this file :-)